addons/payment_stripe/models/payment_provider.py

```diff
@@ -133,6 +133,13 @@ def action_stripe_connect_account(self, menu_id=None):
         """
         self.ensure_one()
 
+        if self.env.company.country_id.code not in const.SUPPORTED_COUNTRIES:
+            raise RedirectWarning(
+                _('Your country is not supported by STRIPE. Please select another payment provider.'),
+                self.env.ref('payment.action_payment_provider').id,
+                _("Other Payment Providers"),
+            )
```

Comment:
Please copy-paste the following:
```suggestion
        if self.env.company.country_id.code not in const.SUPPORTED_COUNTRIES:
            raise RedirectWarning(
                _(
                    "Stripe Connect is not available in your country, please use another payment"
                    " provider."
                ),
                self.env.ref('payment.action_payment_provider').id,
                _("Other Payment Providers"),
            )
```

The message is the same as in `website_sale`'s settings.

Reply:
Hello @AntoineVDV 
Changes have been done, Kindly review them.
Thanks.